src/main/java/view/LottoGameScreen.java

```diff
@@ -0,0 +1,85 @@
+package view;
+
+import domain.ball.LottoBall;
+import domain.lotto.LottoTicket;
+import domain.result.LottoRank;
+import util.OutputUtil;
+import view.dto.LottoGameResultDto;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class LottoGameScreen {
+    public static final String LOTTO_PREFIX = "[";
+    public static final String LOTTO_POSTFIX = "]";
+    public static final String DELIMITER = ", ";
+    public static final String WINNING_LOTTO_CONFIRMATION = "\n지난 주 당첨 번호를 입력해 주세요.";
+    public static final String BONUS_LOTTO_CONFIRMATION = "보너스 볼을 입력해 주세요.";
+    public static final String RESULT = "당첨통계";
+    public static final String LINE = "----------";
+    public static final String LOTTO_RESULT_MESSAGE = "%s개 일치 (%d원)- %d개";
+    public static final String SECOND_PRIZE_RESULT_MESSAGE = "%d개 일치, 보너스 볼 일치 (%d)원- %d";
+    public static final String REVENUE_RESULT_FORMATTER = "총 수익률은 %.2f입니다.";
+
+    public void showAllLottoStatus(final List<LottoTicket> lottoTickets) {
+        lottoTickets.stream()
+                .forEach(lottoTicket -> showTicketStatus(lottoTicket));
```

Comment:
```suggestion
        lottoTickets.forEach(this::showTicketStatus);
```
[Stream의 foreach 와 for-loop 는 다르다](https://woowacourse.github.io/javable/post/2020-05-14-foreach-vs-forloop/)
위 글을 추천드려요! 😉

Reply:
Stream.forEach()는 최종연산이기 때문에 동시성을 보장받기 힘들고 쓰레드에 안전하지 않습니다. 또한. 말 그대로 최종연산이기 때문에 로직이 추가되는것은 이상하다 볼 수 있습니다. 
Coleections.forEach는 Stream객체를 새로 만들고 버리는 과정이 생략되기 때문에 성능향상을 이끌어 낼 수 있고, 또한 내부적으로  synchronized키워드가 있기 때문에 쓰레드 세이프합니다. 

코니의 피드백 덕분에 애매했던 지식을 더 잘 알아가게 되었습니다. 감사합니다.